HTML/Default.sublime-keymap

```diff
@@ -0,0 +1,10 @@
+[
+	{ "keys": [">"], "command": "move", "args": {"by": "characters", "forward": true}, "context":
+		[
+			{ "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
+			{ "key": "following_text", "operator": "regex_contains", "operand": "^>", "match_all": true },
+			{ "key": "selector", "operator": "equal", "operand": "punctuation.definition.tag.end.html", "match_all": true },
+			{ "key": "selector", "operator": "equal", "operand": "text.html.basic" }
```

Comment:
I'd propably not limit that to `.basic` to also enable that in extended syntaxes such as `text.html.jinja` or `text.html.embedded.whatever`? 

IMHO the previous selector `punctuation.definition.tag.end.html` is accurate enough to make it match in appropriate situations only.

Reply:
```yaml
%YAML 1.2
---
# https://www.sublimetext.com/docs/syntax.html
name: HTML (jinja2)
scope: text.html.jinja2

file_extensions:
  - html

contexts:
  main:
    - match: ''
      set:
        - include: scope:text.html.basic
      with_prototype:
        - match: '{%'
          scope: punctuation.section.jinja2.begin.html
          embed: scope:source.python
          embed_scope: source.jinja2.embedded
          escape: '%}'
          escape_captures:
            0: punctuation.section.jinja2.end.html
        - match: ({{)(.+?)(}})
          scope: source.jinja2.embedded
          captures:
            1: punctuation.definition.placeholder.begin.html
            2: variable.parameter.placeholder.html
            3: punctuation.definition.placeholder.end.html
```

Reply:
I've incorporated this improvement